mw-webapp/src/service/Report.ts

```diff
@@ -40,44 +41,17 @@ export class ReportService {
 
 
     // Get a key for a new Post. Generate a random key
-    const newPostKey = push(child(ref(db), "/")).key;
+    const newPostKey: string = push(child(ref(db), "/")).key!;
```

Comment:
waht happend here?

Reply:
If I don't use ! symbol newPostKey has type string | null . But key mustn't be null for create new report.